examples/incito-publication-template.html

```diff
@@ -66,7 +66,16 @@
             data-component-publication-disable-global-scrollbar="false"
             data-translation-keys--publication-viewer-shopping-list-label="Shopping List"
             data-translation-keys--publication-viewer-download-button="Download Now"
-            defer
```

Comment:
This is no go, IMO. We need to defer so our SDK doesn't block the entire website.